nodes/index.adoc

```diff
@@ -129,3 +129,53 @@ As an administrator, You can perform various tasks on a Linux container, such as
 
 Apart from performing specific tasks on nodes, pods, and containers, you can work with the overall {product-title} cluster to keep the cluster efficient and the application pods highly available.
+
+[id="commonterms-node"]
+== Common terms
+
+This glossary defines common terms that are used in the _node_ content. These terms help you understand _nodes_ effectively.
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere, from a data center to a public or private cloud to even a developer's laptop.
+
+Daemon set::
+It ensures that a replica of the pod runs on eligible nodes in an {product-title} cluster.
+
+egress::
+Is outgoing traffic from a pod. It is the process of data sharing externally through a network’s outbound traffic.
+
+garbage collection::
+It is the process of cleaning up cluster resources, such as terminated containers and images that are not referenced by any running pods.
+
+Horizontal Pod Autoscaler(HPA)::
+Implemented as a Kubernetes API resource and a controller. You can use the HPA to specify the minimum and maximum number of pods you want to run. You can also specify the CPU or memory utilization that your pods should target.
+
+Ingress::
+Is incoming traffic to a pod.
+
+Job::
+A process that runs to completion. A job creates one or more pod objects and ensures that the specified pods are successfully completed.
+
+Labels::
+Is a key value pair attached to an object such as a pod. You can use labels to organize and select subsets of objects.
+
+Node::
+A worker machine in the {product-title} cluster. A node can be either be a virtual machine (VM) or a physical machine.
+
+Node Tuning Operator::
+You can use the Node Tuning Operator to manage node-level tuning by using the TuneD daemon. It ensures custom tuning specifications are passed to all containerized TuneD daemons running in the cluster in the format that the daemons understand.
+The daemons run on all nodes in the cluster, one per node.
+
+Self Node Remediation Operator::
+The Operator runs on the cluster nodes and identifies and reboots nodes that are unhealthy.
+
+Pod::
+Is one or more containers with shared resources, such as volume and IP addresses, running in your {product-title} cluster.
+A pod is the smallest compute unit defined, deployed, and managed.
+
+Toleration::
+Indicates that the pod is allowed (but not required) to be scheduled on nodes or node groups with matching taints. You can use tolerations to enable the scheduler to schedule pods with matching taints.
+Taints and tolerations work together to ensure that pods are not scheduled on irrelevant nodes.
```

Comment:
What do you think about removing this last sentence from the toleration glossary definition since you already have that sentence under the taint glossary definition?